plugin/trino-phoenix5/src/main/java/io/trino/plugin/phoenix5/TypeUtils.java

```diff
@@ -164,7 +175,7 @@ private static Object jdbcObjectToTrinoNative(ConnectorSession session, Object j
         }
 
         if (REAL.equals(type)) {
-            return floatToRawIntBits((float) jdbcObject);
+            return ((Integer) floatToRawIntBits((float) jdbcObject)).longValue();
```

Comment:
`floatToRawIntBits` returns you an integer, so you need just a cast to long:

```suggestion
            return (long) floatToRawIntBits((float) jdbcObject);
```

-- shorter & no boxing needed